controllers/routemap.go

```diff
@@ -22,19 +22,22 @@ import (
 
 /* types: begin */
 
-type routemap struct {
+// Routemap identifies a set of versions that comprise and application or ML model
+// and associates them with a set of routing templates
+type Routemap struct {
```

Comment:
Create `interface.go` with the following interfaces.

```go
type RouteMapsInt interface {
// with the right methods
}

type RouteMapInt interface {
// with the right methods
}

type VersionInt interface {
// with the right methods
}
```

Reply:
done